core/runtime/asm/asm_riscv32.S

```diff
@@ -0,0 +1,58 @@
+.section .start, "ax"
+.globl start
+start:
```

Comment:
Could we add a comment describing the arguments?

Reply:
Also probably worth commenting what `rt_header` offsets are, just to make things clear

Reply:
I added a top-of-file comment describing `rt_header` and a "function comment" describing the arguments.